cmd/sink/main.go

```diff
@@ -55,13 +61,26 @@ func NewSink(db database.DBInterface, logger *log.Logger) *Sink {
 		logger.Fatalf("failed to create CloudEvents HTTP client: %s\n", err.Error())
 	}
 
+	deleteWhen := os.Getenv(DeleteWhenEnvVar)
+	deleteWhen, err = jsonpath.RelaxedJSONPathExpression(deleteWhen)
+	if err != nil {
+		logger.Fatalf("Provided JSON Path %s could not be parsed: %s", deleteWhen, err)
```

Comment:
I think you need a '\n' here as well:

```suggestion
		logger.Fatalf("Provided JSON Path %s could not be parsed: %s\n", deleteWhen, err)
```

Reply:
done